src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -1434,7 +1434,9 @@ private void initActions() {
         twoEntriesOnlyActions.addAll(Arrays.asList(mergeEntries));
 
         atLeastOneEntryActions.clear();
-        atLeastOneEntryActions.addAll(Arrays.asList(downloadFullText, lookupIdentifiers));
+        atLeastOneEntryActions.addAll(Arrays.asList(downloadFullText));
+        atLeastOneEntryActions.clear();
```

Comment:
@tobiasdiez sorry for the confusion and thanks for the note. I misunderstood something while doing try and error. I've reverted the changes.